string room_name
string actuator_name
string status
---
